oidc-service-impl/src/main/java/io/mosip/esignet/services/OAuthServiceImpl.java

```diff
@@ -91,9 +90,17 @@ public class OAuthServiceImpl implements OAuthService {
     @Value("${mosip.esignet.discovery.issuer-id}")
     private String discoveryIssuerId;
 
+    @Value("${mosip.esignet.par.expire-seconds:60}")
+    private int parTTLInSeconds;
+
+    @Value("${mosip.esignet.par-prefix:}")
```

Comment:
change the property name 
"mosip.esignet.par.request-uri.prefix" with default value as "urn:ietf:params:oauth:request_uri:"